test/expect/TestScript.test_lstm_fusion_cuda-backward.expect

```diff
@@ -0,0 +1,87 @@
+graph(%0 : Float(3, 20!)
+      %1 : Float(3, 20!)
+      %2 : Dynamic
+      %3 : Dynamic
+      %4 : Dynamic
+      %5 : Dynamic
+      %6 : Dynamic
+      %7 : Dynamic
+      %8 : Dynamic
+      %x : Float(3, 10)
+      %hx : Float(3, 20)
+      %cx : Float(3, 20)
+      %12 : Float(10!, 80!)
+      %13 : Float(20!, 80!)
+      %ingate : Float(3, 20)
+      %forgetgate : Float(3, 20)
+      %cellgate : Float(3, 20)
+      %outgate : Float(3, 20)
+      %18 : Float(3, 20)) {
+  %19 : int = prim::Constant[value=1]()
+  %20 : Float(3, 20) = prim::FusionGroup_0[device=0](%18)
+  %21 : Float(3, 20) = aten::add(%20, %19, %19)
+  %22 : Float(3, 20) = prim::FusionGroup_1[device=0](%1, %21, %0, %outgate)
+  %23 : Float(3, 20) = aten::mul(%22, %19)
+  %24 : Float(3, 20) = aten::neg(%outgate)
+  %25 : Float(3, 20) = aten::add(%24, %19, %19)
+  %26 : Float(3, 20) = prim::FusionGroup_2[device=0](%cellgate)
+  %27 : Float(3, 20) = aten::add(%26, %19, %19)
+  %28 : Float(3, 20) = aten::neg(%forgetgate)
+  %29 : Float(3, 20) = aten::add(%28, %19, %19)
+  %30 : Float(3, 20) = aten::neg(%ingate)
+  %31 : Float(3, 20) = aten::add(%30, %19, %19)
+  %32 : Float(3, 80) = prim::FusionGroup_3[device=0](%31, %ingate, %29, %forgetgate, %27, %25, %outgate, %22, %cx, %23, %cellgate, %0, %18)
+  %33 : Float(3, 80) = aten::mul(%32, %19)
+  %34 : Float(80!, 3!) = aten::t(%33)
+  %35 : Float(80, 20) = aten::mm(%34, %hx)
+  %36 : Float(80!, 3!) = aten::t(%32)
+  %37 : Float(80, 10) = aten::mm(%36, %x)
+  return (%37, %35, %33, %33);
+}
```

Comment:
Uh so many missed fusion opportunities 😄Looking forward to your patches landing.